client/app/components/admin/CeleryStatus.jsx

```diff
@@ -46,7 +46,14 @@ function parseTasks(tasks) {
 function QueuesTable({ loading, queues }) {
```

Comment:
Perhaps an opportunity to fix the "forEach" error thrown when there're no celery tasks.
```js
function parseTasks(tasks = []) {
```